neptune/internal/backends/alpha_integration_backend.py

```diff
@@ -446,21 +447,108 @@ def _get_client_config_args(api_token):
+                            ) -> List[alpha_exceptions.MetadataInconsistency]:
+        experiment_uuid = uuid.UUID(experiment.internal_id)
+        kwargs = {
+            'experimentId': str(experiment_uuid),
+            'operations': [
+                {
+                    'path': alpha_path_utils.path_to_str(op.path),
+                    AlphaOperationApiNameVisitor().visit(op): AlphaOperationApiObjectConverter().convert(op)
+                }
+                for op in operations
+            ]
+        }
         try:
-            errors = self._alpha_backend.execute_operations(
-                experiment_uuid=uuid.UUID(experiment.internal_id),
-                operations=operations
-            )
-            if errors:
-                raise AlphaOperationErrors(errors)
-        except alpha_exceptions.ExperimentUUIDNotFound as e:
+            result = self.leaderboard_swagger_client.api.executeOperations(**kwargs).response().result
+            return [
+                alpha_exceptions.MetadataInconsistency(err.errorDescription)
+                for err in result
+            ]
+        except HTTPNotFound as e:
             # pylint: disable=protected-access
             raise ExperimentNotFound(
-                experiment_short_id=experiment.id, project_qualified_name=experiment._project.full_id) from e
-        except alpha_exceptions.InternalClientError as e:
-            raise NeptuneException(e) from e
+                experiment_short_id=experiment.id,
+                project_qualified_name=experiment._project.full_id
+            ) from e
+
+    def _execute_alpha_operation(self, experiment: Experiment, operations: List[alpha_operation.Operation]):
+        experiment_uuid = uuid.UUID(experiment.internal_id)
+        errors = []
+
+        operations_preprocessor = AlphaOperationsPreprocessor()
+        operations_preprocessor.process(operations)
+        errors.extend(operations_preprocessor.get_errors())
+
+        upload_operations, other_operations = [], []
+        file_operations = (
+            alpha_operation.UploadFile,
+            alpha_operation.UploadFileContent,
+            alpha_operation.UploadFileSet
+        )
+        for op in operations_preprocessor.get_operations():
+            (upload_operations if isinstance(op, file_operations) else other_operations).append(op)
+
+        # Upload operations should be done first since they are idempotent
+        errors.extend(
+            self._execute_upload_operations(experiment_uuid=experiment_uuid,
+                                            upload_operations=upload_operations)
+        )
+
+        if other_operations:
+            errors.extend(
+                self._execute_operations(experiment=experiment,
+                                         operations=other_operations)
+            )
+
+        if errors:
+            raise AlphaOperationErrors(errors)
+
+        return None
```

Comment:
Isn't it an overkill to copy all this logic here? Is there even a case when you need to perform upload operation and simple operation? These logic was necessary in alpha backend because we do not know what operations will be passed. But in this class you exactly know what do you want to perform:
upload_file_set_attribute - when uploading source code
upload_file_attribute - in case of artifacts
_execute_operations - in case of simple operations